blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -75,15 +75,15 @@ export class Activity extends Component {
         const head = 'pipelinedetail.activity.header';
 
         const status = t(`${head}.status`, { defaultValue: 'Status' });
-        const build = t(`${head}.build`, { defaultValue: 'Build' });
+        const run = t(`${head}.run`, { defaultValue: 'Run' });
```

Comment:
@scherler this sounds perfect. But if "i18n services are down" that means "Jenkins is down" and we have bigger problems, I don't think it makes any sense to have defaults like this.

@imeredith 🐜  despite the shorter bits of typing, I really don't like the `const head = ...`, worse for maintainability, can't search for strings in the codebase and doing actual javascript work for no real reason...